catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -101,6 +101,7 @@
     <feature name="offline-gazetteer" version="${project.version}"
              description="Offline gazetteer service utilizing a local GeoNames index.">
         <feature>catalog-solr-provider</feature>
+        <bundle>mvn:org.codice.countrycode/converter/${countryconverter.version}</bundle>
```

Comment:
I think eventually the converter will be used elsewhere; might be worthwhile to move this to the spatial-core or spatial-app feature

Reply:
It needs to happen before the `<bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-create/${project.version}</bundle>`, but I will add to `spatial-app` as well